br/backup-and-restore-overview.md

```diff
@@ -15,6 +15,32 @@ TiDB 备份恢复功能可以用于满足以下业务的需求：
 - 审计业务的历史数据，满足司法审查的需求。
 - 复制 (Clone) 生产环境，方便问题诊断、性能调优验证、仿真测试等。
 
+## 使用须知
+
+本部分介绍使用 TiDB 备份恢复功能前的注意事项，包括使用限制和使用建议。
+
+### 使用限制
+
+- PITR 仅支持恢复到**全新的空集群**。
+- PITR 仅支持集群粒度的恢复，不支持对单个 database 或 table 的恢复。
+- PITR 不支持恢复用户表和权限表的数据。
+
```

Comment:
```suggestion
PITR 恢复集群的任务运行期间，不支持打开日志备份任务，也不支持通过 TiCDC 同步数据到下游集群。
```

Reply:
I'll create another PR to add this sentece. Please remove the request change label. Thanks.